backends/magma/ceed-magma.h

```diff
@@ -13,6 +13,34 @@
 #include <ceed/backend.h>
 #include <magma_v2.h>
 
+#define MAGMA_MAXTHREADS_1D 128
+#define MAGMA_MAXTHREADS_2D 128
+#define MAGMA_MAXTHREADS_3D 64
+// Define macro for determining number of threads in y-direction
+// for basis kernels
+#define MAGMA_BASIS_NTCOL(x, maxt) (((maxt) < (x)) ? 1 : ((maxt) / (x)))
+// Define macro for computing the total threads in a block
+// for use with __launch_bounds__()
+#define MAGMA_BASIS_BOUNDS(x, maxt) (x * MAGMA_BASIS_NTCOL(x, maxt))
+
+#ifdef CEED_MAGMA_USE_HIP
+#define CeedMagmaModule hipModule_t
+#define CeedMagmaFunction hipFunction_t
+#define CeedCompileMagma CeedCompileHip
+#define CeedGetKernelMagma CeedGetKernelHip
+#define CeedRunKernelMagma CeedRunKernelHip
+#define CeedRunKernelDimMagma CeedRunKernelDimHip
+#define CeedRunKernelDimSharedMagma CeedRunKernelDimSharedHip
+#else
+#define CeedMagmaModule CUmodule
+#define CeedMagmaFunction CUfunction
+#define CeedCompileMagma CeedCompileCuda
+#define CeedGetKernelMagma CeedGetKernelCuda
+#define CeedRunKernelMagma CeedRunKernelCuda
+#define CeedRunKernelDimMagma CeedRunKernelDimCuda
+#define CeedRunKernelDimSharedMagma CeedRunKernelDimSharedCuda
+#endif
```

Comment:
```suggestion
#ifdef CEED_MAGMA_USE_HIP
#typedef CeedMagmaModule hipModule_t
#typedef CeedMagmaFunction hipFunction_t
#define CeedCompileMagma CeedCompileHip
#define CeedGetKernelMagma CeedGetKernelHip
#define CeedRunKernelMagma CeedRunKernelHip
#define CeedRunKernelDimMagma CeedRunKernelDimHip
#define CeedRunKernelDimSharedMagma CeedRunKernelDimSharedHip
#else
#typedef CeedMagmaModule CUmodule
#typedef CeedMagmaFunction CUfunction
#define CeedCompileMagma CeedCompileCuda
#define CeedGetKernelMagma CeedGetKernelCuda
#define CeedRunKernelMagma CeedRunKernelCuda
#define CeedRunKernelDimMagma CeedRunKernelDimCuda
#define CeedRunKernelDimSharedMagma CeedRunKernelDimSharedCuda
#endif
```

these two are types, right?

Reply:
Yeah, whoops, fixed now